src/components/LayoutMobile.js

```diff
@@ -69,7 +70,9 @@ export default function Layout({ children }) {
         >
           <App />
         </Box>
-        <Drawer>{children}</Drawer>
+        <Drawer>
+          <Box ref={ref}>{children}</Box>
```

Comment:
Can we forward this ref to the `Drawer` so we can target the element that hold the `children`. Currently it does not scroll all the way to the top, there is still a "gap"

Reply:
done